### PR TITLE
Adding security alert setting to setting search

### PR DESCRIPTION
**Pull request by jpuri**
## **Description**
Fix the issue that PPOM security alerts are not searchable in settings.

## **Manual testing steps**

1. Go to settings page
2. Search for "Security Alerts"
3. It should show search result
4. Click the search result and it should take you to experimental page

## **Screenshots/Recordings**
Before:
Screenshot 2023-10-03 at 6 41 02 PM

After:
Screenshot 2023-09-28 at 7 42 36 PM

## **Related issues**

_Fixes #20753

## **Pre-merge author checklist**

- [x] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/CODING_GUIDELINES.md).
- [x] I've clearly explained:
  - [ ] What problem this PR is solving.
  - [ ] How this problem was solved.
  - [x] How reviewers can test my changes.
- [x] I’ve indicated what issue this PR is linked to: Fixes #???
- [ ] I’ve included tests if applicable.
- [x] I’ve documented any added code.
- [x] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/LABELING_GUIDELINES.md)).
- [x] I’ve properly set the pull request status:
  - [ ] In case it's not yet "ready for review", I've set it to "draft".
  - [ ] In case it's "ready for review", I've changed it from "draft" to "non-draft".

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

**Comment by jpuri** [comment]
@bschorchit : can you plz confirm if this search string `Security Alerts` looks good ?

Screenshot 2023-09-28 at 7 42 36 PM

**Review by digiwand — DISMISSED** [review]
Nice! ~might be handy to mention we're disabling the experimental route for desktop (or other relevant builds I haven't looked into)~

File: ui/helpers/constants/settings.js [hunk 1]
```diff
@@ -7,7 +7,9 @@ import {
   ABOUT_US_ROUTE,
   NETWORKS_ROUTE,
   CONTACT_LIST_ROUTE,
+  ///: BEGIN:ONLY_INCLUDE_IN(build-main,build-beta,build-mmi,blockaid)
   EXPERIMENTAL_ROUTE,
+  ///: END:ONLY_INCLUDE_IN
```
Comment (digiwand) [inline ui/helpers/constants/settings.js:12]: can we add this update in the description and title as well?

**Comment by jpuri** [comment]
> Nice! might be handy to mention we're disabling the experimental route for desktop (or other relevant builds I haven't looked into)

@digiwand : experimental route will continue to work as before, desktop includes flask - thus it will work on desktop also.

**Comment by digiwand** [comment]
@jpuri oh. that makes sense. okay please dismiss my comment about the desktop build now. I'm still confused about the purpose of this particular code-fence.

```
///: BEGIN:ONLY_INCLUDE_IN(build-main,build-beta,build-mmi,blockaid)
  EXPERIMENTAL_ROUTE,
  ///: END:ONLY_INCLUDE_IN
```

can you please help explain?

**Comment by digiwand** [comment]
I see. so the code fencing has been reverted 👍🏼  I was confused about the desktop build... thank you for explaining

**Review by digiwand — COMMENTED** [review]
can we include a test here as well? `ui/helpers/utils/settings-search.test.js`

**Review by brad-decker — DISMISSED** [review]

**Review by segun — DISMISSED** [review]

**Review by digiwand — DISMISSED** [review]

**Review by segun — APPROVED** [review]

**Review by blackdevelopa — APPROVED** [review]

**Pull request merged**